graphene/core/schema.py

```diff
@@ -69,8 +67,7 @@ def T(self, _type):
     @property
     def executor(self):
         if not self._executor:
-            self._executor = Executor(
-                [SynchronousExecutionMiddleware()], map_type=OrderedDict)
+            self._executor = SyncExecutor()
```

Comment:
This is the only part that seems to have drastically changed - I did some investigating and it looks like the Executor uses an OrderedDict by default now so I don't think I needed to specify it explicitly.